Cargo.toml

```diff
@@ -43,5 +44,5 @@ tokio = { version = "1.26.0", features = ["full"] }
 toml = "0.7.2"
 tower-http = { version = "0.4.0", features = ["cors"] }
 
-[dev-dependencies]
-serial_test = "1.0.0"
+# [dev-dependencies]
+# serial_test = "2.0.0"
```

Comment:
We don't need it at the moment, but it's there in case it is needed. It sets up a test so that it's guaranteed to run sequentially. See:
https://docs.rs/serial_test/latest/serial_test/